nixos/doc/manual/release-notes/rl-2411.section.md

```diff
@@ -19,6 +19,8 @@
 - `openssh` and `openssh_hpn` are now compiled without Kerberos 5 / GSSAPI support in an effort to reduce the attack surface of the components for the majority of users. Users needing this support can
   use the new `opensshWithKerberos` and `openssh_hpnWithKerberos` flavors (e.g. `programs.ssh.package = pkgs.openssh_gssapi`).
 
+- The `openssh` module now uses upstream's default cryptographic algorithms, as part of a broad effort to reduce the upstream diff in response to the 2024 xz incident.
```

Comment:
```suggestion
- The `openssh` module now uses upstream's default cryptographic algorithms in an effort to reduce required maintenance.
```
I think mentioning xz here will just trigger unnecessary questions and lead people into a wrong direction.